tests/foreman/ui_airgun/test_lifecycleenvironment.py

```diff
@@ -254,6 +256,54 @@ def test_positive_search_lce_content_view_packages_by_name(
                 assert entry['Name'].startswith(package['name'])
 
 
+@tier3
+def test_positive_search_lce_content_view_module_streams_by_name(
+        session, module_org):
+    """Search Lifecycle Environment content view module streams by name
+
+    :id: e67893b2-a56e-4eac-87e6-63be897ba912
+
+    :customerscenario: true
+
+    :steps:
+        1. Create a product with a repository synchronized
+            - The repository must contain at least two module stream names P1 and
+              P2
+            - P1 has two module streams
+            - P2 has three module streams
+        2. Create a content view with the repository and publish it
+        3. Go to Lifecycle Environment > Library > ModuleStreams
+        4. Select the content view
+        5. Search by module stream names
+
+    :expectedresults: only the searched module streams where found
+
+    :BZ: 1432155
```

Comment:
1432155 talks about packages, not about module streams, so I would remove:
```
:BZ: 1432155
```
and
```
:customerscenario: true
```

Reply:
yeah copy+paste mistake